openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -216,6 +219,8 @@ def celebrations(self):
         """
         return {
             'first_section': CourseEnrollmentCelebration.should_celebrate_first_section(self.enrollment_object),
+            'first_discussion': UserCelebration.should_celebrate_first_discussion(self.effective_user, self.course_key),
+            'first_discussion_user_bucket': COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES_FIRST_DISCUSSION_POST.get_bucket(),
```

Comment:
This is used in the frontend to inform
a) if the modal should be shown or if it is the control group (bucket 0)
b) if just the modal with no body text should be shown (bucket 1)
c) or if the modal with the normative data in the body should be shown (bucket 2)